examples/morphir-elm-projects/evaluator-tests/src/Morphir/Examples/App/SdkBasicTests.elm

```diff
@@ -1875,42 +1881,49 @@ basicsAlwaysTest x =
     in
     f [ 4 ]
 
+
 {-| Test: SdkBasics/basicsClampTest
 -}
 basicsClampTest : a -> a -> a -> a
```

Comment:
I'd change all of these type variables from `a` to `number`... IDK if it's the intention that that be a rigidly enforced in morphir, but it is in elm, and enforced or not is the only way of indicating that the type must belong to the number typeclass.

Reply:
i can do this in a separate pr since this was just part of running the formatter

Reply:
The formatter changed `number` to `a`?

Reply:
Oh wait, nvm I see what you meant, sure